projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/timepicker-docs/timepicker-docs.component.html

```diff
@@ -0,0 +1,309 @@
+<section class="go-container">
+    <app-form-control-docs class="go-column go-column--100"></app-form-control-docs>
+  
+    <go-card id="form-input-control" class="go-column go-column--100">
+      <ng-container go-card-header>
+        <h2 class="go-heading-2 go-heading--no-wrap">Component Control</h2>
+        <go-copy cardId="form-input-control" goCopyCardLink></go-copy>
+      </ng-container>
+      <div class="go-container" go-card-content>
+  
+        <div class="go-column go-column--100">
+          <p class="go-body-copy go-body-copy--no-margin">
+            As with most of the form components, the only <code class="code-block--inline">@Input</code>
+            required of the timepicker component is a <code class="code-block--inline">FormControl</code>.
+            The FormControl can be passed in via the <code class="code-block--inline">[control]</code>
+            attribute on the &lt;go-timepicker&gt; component. A time can be initialized into the timepicker
+            by setting a time on the control. It can be initialized by either a string or a date object.
```

Comment:
This states that the control can be initialized by setting a date object. Is this true? If so can we add a test for this in the `go-timepicker.component.spec.ts` tests?